packages/rabbitmq/src/amqp/connection.ts

```diff
@@ -77,7 +78,8 @@ export class AmqpConnection {
 
     this.publish(requestOptions.exchange, requestOptions.routingKey, payload, {
       replyTo: DIRECT_REPLY_QUEUE,
-      correlationId
+      correlationId,
+      ...publishOptions
```

Comment:
The signature for `publish` is:
```
public publish(
    exchange: string,
    routingKey: string,
    message: any,
    options?: amqplib.Options.Publish
  )
```
So it's expecting the options as a 4th arg.  Looks like you're spreading them into the 3rd (message) arg here so I don't think it'll interact properly with the underlying amqplib

Reply:
let me double check I did this right...

Reply:
Whoops nevermind I misread this

Reply:
Yes, I wanted to keep the values that were specific to request but allow additional items to be added.